lib/dpla/map/factories.rb

```diff
@@ -10,7 +10,7 @@
     date { |timespan| timespan.association :timespan, :strategy => :build }
     description 'Window of the Stonewall Bar N.Y. 1969. The other half of the graffiti was erased by the time Diana photographed it.'
     extent '10x12 cm'
-    format 'Silver Gelatin Print'
+    dcformat 'Silver Gelatin Print'
```

Comment:
Other option would be to undefine conflicting methods within the DSL's scope. I'm leaning this way, since it reopens the property namespace. Need to think on what the options are and what the implications of doing this would be.